src/sql/Expr.h

```diff
@@ -21,6 +21,11 @@ enum ExprType {
   kExprLiteralNull,
   kExprLiteralDate,
   kExprLiteralInterval,
+  kExprLiteralTimestamp,
+  kExprLiteralTime,
+  kExprCurrentTimestamp,
+  kExprCurrentDate,
+  kExprCurrentTime,
```

Comment:
We discussed this internally and we prefer to have only an expression type per date/time/timestamp, not for the "current" version of the literal. Instead, please add a (boolean) flag to the `Expr` struct indicating whether the literal has the special "current" value.

Mini: The expression txpes are not really ordered by name. Could you bring them in the order date, time, timestamp, interval?